apps/src/code-studio/pd/application_dashboard/quick_view.jsx

```diff
@@ -127,8 +128,26 @@ export class QuickView extends React.Component {
   };
 
   render() {
+    let accepted = 0;
+    let registered = 0;
+    if (this.state.applications !== null) {
+      accepted = this.state.applications
+        .filter(app => app.status === 'accepted')
+        .length;
+      registered = this.state.applications
+        .filter(app => app.registered_workshop === 'Yes')
+        .length;
+    }
     return (
       <div>
+        {this.state.applications &&
+          <CohortCalculator
+            role={this.props.route.path.replace('_cohort', '')}
```

Comment:
same as above. We should get this from props, not by manipulating the path. Also in this case, in quick_view, there is no `_cohort` part of the path, so this `replace` is a no-op.

Reply:
good point - this is how we get the path in [cohort view](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/pd/application_dashboard/cohort_view.jsx#L78), so I'll fix it in both places.